### DISCUSSION TITLE
Diga não ao Medium: tenha sua própria plataforma

### DISCUSSION THREAD

**Discussion by willianjusten** [frontendbr/forum#1505] [Discussão]
Fala pessoal, já tiveram algumas discussões sobre esse assunto lá no Twitter, mas eu queria trazer para cá, porque gosto bastante da opinião das pessoas daqui e acho legal para ter essa fonte aqui para outras pessoas que estejam chegando na web e no fórum.

Eu escrevi [um post](https://willianjusten.com.br/diga-nao-ao-medium-tenha-sua-propria-plataforma/) falando de algumas das razões do porquê não utilizar o Medium e sim sua própria plataforma.

Alguns dos pontos foram:

1. Ownership do Projeto
2. Medium é uma empresa e pode morrer da noite para o dia.
3. Possibilidade de Personalização
4. Aprendizado
5. UI/UX confusa no Medium
6. Ter um blog opensource

No post eu explico cada ponto, mas para num ficar muito grande aqui, deixo só os pontos. 

Você concorda com isso? Se sim, tem mais alguns pontos a acrescentar?
Você não concorda? Por que não?

**Comment by mahenrique94** [comment]
Concordo plenamente e acrescentaria a parte de cobrança, agora somos limitados a ler X post's por um intervalo de tempo, se estivermos lendo muito, ele trava e pede para assinar.

**Comment by aeusteixeira** [comment]
Queria muito criar um blog estático pelo GitHub Pages, mas não consegui instalar o Ruby em minha máquina para utilizar o Jekyll

**Comment by kavalcante** [comment]
Eu particulamente prefiro ter a sensação de "comunidade" e rede social do Medium. Acho que o poder dele me sugerir artigos conforme o que eu costumo ler me poupa bastante tempo e me faz ler mais. Coisa que em um blog pessoal, eu dificilmente faria.

Sobre as discussões da plataforma ser paga e etc... Acho que faz parte. Pelo o que eu entendi usando a ferramenta, eu tenho um limite mensal de leitura e o member preview pareceu ser uma opção do autor. Se for este o caso, acho até bem justo.

De qualquer forma, ainda prefiro utilizar o Medium a ter que acompanhar muitos blogs ou ficar atento no Twitter. Pra mim, que não uso muito redes sociais, o Medium me ajuda bastante nesse sentido.

**Comment by kvnol** [comment]
> Queria muito criar um blog estático pelo GitHub Pages, mas não consegui instalar o Ruby em minha máquina para utilizar o Jekyll

@matheustx tenta rodar o jekyll com o Docker, aí tu nem precisa instalar na sua máquina.

https://hub.docker.com/r/jekyll/jekyll/

**Comment by evertonthepaula** [comment]
Sabe eu não tinha pensado nisto ainda, mas acredito que só fez sentido para mim hoje porque uso o Medium para minhas publicações. Na minha visão acredito que os passos naturais seriam:

1 - Escreva e compartilhe
2 - Publique em uma plataforma "grátis" (Medium ou Tableless)
3 - Palestre
4 - Crie sua própria plataforma

Acho que o futuro seria termos uma plataforma no melhor estilo Web Service onde você pode listar todos os blogs da comunidade e clicar para visitar este blog(alguém mais usa pocket aí?). E Para isso acontecer bastando que o dono do blog use a API para registrar seu blog e atualizar as novas publicações.

**Comment by guilhermethales** [comment]
Não consigo opinar muito em usar o Medium como escritor, pois nunca fiz isso, mas creio que as pessoas gostem por ser fácil de utilizar e de não ter que se preocupar com nada além de escrever o post e publicar. Às vezes a pessoa acha bacana tudo isso de aprendizado, open-source, etc, mas ela apenas quer escrever seus posts e compartilhar conteúdo de forma fácil.

Concordo com quase todos os itens que você citou, tirando o item Design UI/UX, pois eu gosto bastante, realmente não creio que é um item que estaria como uma desvantagem do Medium e uma possível vantagem de algum blog pessoal. Provavelmente o Medium deve ter um UI/UX melhor que 95%(modo de falar) dos blogs pessoais, ou pelo menos dos que eu já pude ver.

Agora, como leitor, eu gosto bastante do Medium, consigo acessar conteúdo de altíssimo nível do mundo inteiro de forma muito simples, assim como interagir no post caso necessário. Também não tenho que ficar vendo anúncios enchendo o saco a todo momento como em diversos locais, e quanto às críticas à plataforma começar a cobrar, não consigo entender muito a indignação com isso. Qualquer um continua tendo acesso de forma ilimitada aos conteúdos de pessoas que não DECIDEM ser parceiros do Medium. Você tem uma limitação de 3 posts apenas para conteúdos de parceiros do Medium, e mesmo que não fosse assim, seria uma estratégia de Marketing de uma empresa assim como qualquer outra.

Enfim, acho extremamente válido tudo o que disse quanto à criar um blog e deve ser uma experiência muito boa, inclusive pretendo criar o meu ainda, rs. Mas não acho que seja necessário PARAR de usar o Medium. Por que não utilizar os dois?

**Comment by fabiomirandaa** [comment]
> Queria muito criar um blog estático pelo GitHub Pages, mas não consegui instalar o Ruby em minha máquina para utilizar o Jekyll

Você pode usar o [Gatsby](https://www.gatsbyjs.org/)

**Comment by kazzkiq** [comment]
A única vantagem que eu vejo no Medium é a visibilidade que ele proporciona. Se você não for relevante nas redes sociais, palestrante ou algo do tipo, dificilmente seus posts vão ter visibilidade e aí seu blog terá um crescimento ridiculamente lento (a vantagem disso é que o público, mesmo pequeno, acaba se tornando fiel). 

No Medium você acaba tendo mais alcance, porém o custo é a centralização de conteúdo na plataforma.

Eu particularmente sou contra qualquer tipo de centralização na web. Acho que quanto mais sites independentes tivermos, mais saudável a web será.

E pra nós que somos devs, temos ainda menos desculpas pra criar nossa própria plataforma!

**Comment by AdrianoLupossa** [comment]
Existem muitas vantagens em ter um blog pessoal, mas para algumas pessoas que simplesmente desejam escrever e publicar rapidamente o medium é muito viável, pois nem todo mundo quer ter o "trabalho" de criar, dar manutenção,  fazer optimizações de SEO, etc.

**Comment by AdrianoLupossa** [comment]
Acho que na web mesmo descentralizada sempre iremos precisar de algum tipo de agregador de conteúdos, porque há muita informação e por causa disso sempre vai haver algum tipo de "monopólio".

Basicamente tanto o google ou o medium praticamente são agregadores de conteúdo.

**Comment by ViniciusGularte** [comment]
Estou migrando para o [dev.to](https://dev.to) que é open source e tem ótimos artigos também :smile: , pessoal do [Freecodecamp](https://www.freecodecamp.org) também caiu fora do medium, eu tinha submetido mais um artigo pro pessoal e o Quincy Larson(Fundador do freecodecamp) me disse que eles sairam por causa da pressão do medium pra colocar artigos pagos(Coisa que o Freecodecamp não faz :angel: ),  o que eu recomendo é criar seu blog pessoal, e acho que não tem problema também colocar no medium, e em outras plataformas, quanto mais compartilhar conteúdo melhor, compartilha conteudo em outras plataformas fazendo referência ao seu blog(Já colabora com o SEO do site :rocket: ).

**Comment by willianjusten** [comment]
Pessoal tá falando sobre agregador de conteúdo e de precisar ficar seguindo e etc. Vocês já ouviram falar em RSS? Antes, nós usávamos apps para agregar todo o conteúdo que nós queriamos e de forma bem fácil. 

Existe o [Feedly](https://feedly.com/), [InoReader](https://www.inoreader.com/) e outros mais, onde podemos dizer quais blogs queremos seguir e ele nos avisa se tiver post novo. O próprio [Pocket](https://getpocket.com/) também dá links recomendados baseado naquilo que você adiciona.

---

Sobre a parte de visibilidade, isso funciona muito em partes, muita gente fala "Post do Medium" e não "Post do fulano", então você acha que está bombando, sendo que na realidade, quem está bombando é o Medium, que além de ganhar acessos, ganha dados de todos os usuários que você está atraindo...

**Comment by willianjusten** [comment]
Sobre dizer que é difícil/chato criar um blog próprio e talz, existem até CLIs que importam os posts para você, [export-medium-to-gatsby](https://github.com/jamischarles/export-medium-to-gatsby)

**Comment by evertonthepaula** [comment]
O problema desse modelo é que você ainda tem que encontrar o conteúdo da pessoa pra então começar a seguir a pessoa, isto quer dizer ir de site em site, em 480 lugares diferentes, para dizer que quer seguir o conteúdo daquela pessoa. Quando se tem uma plataforma o conhecimento está lá, quase como em uma biblioteca.

**Comment by isacmoura** [comment]
Como já mencionaram acima, eu acredito que a vantagem do Medium é exatamente o "escreva e publique". Para mim que ainda estou na faculdade pagar um domínio não dá e GitHub Pages não é tão acessível para quem não é da área. A desvantagem em questão dos anúncios pagos foi o que matou o Medium na minha opinião.

Como sugeriram acima também, eu acredito que o dev.to pode ser uma excelente alternativa para quem não deseje ter que criar um blog no momento e só se preocupar com o "escreva e publique".

**Comment by vhfmag** [comment]
Concordo plenamente, até fiz o mesmo! :blush: E kudos pela recomendação de RSS, também adotei (inclusive, encontrei seu post por lá) e agora posso acompanhar o ritmo frenético da comunidade web sem ficar pendurado no Medium e no Twitter, além de ter uma lista de feeds que é minha, posso exportar e fazer backup ou mudar de agregador sem medo de perder nada por mudar de plataforma ~(ou por uma plataforma ou governo decidir me deixar de fora)~.

Quanto aos geradores de sites estáticos (ou SSG), quero adicionar à sugestão do Gatsby (que já usei e é excelente, mas gera Javascript demais pra blogs, que costumam ter pouca ou nenhuma interatividade) o [Eleventy](https://11ty.io), minha escolha atual – dá pra usar inúmeros formatos de template (Nunjucks, Markdown, Liquid, Pug, Javascript...), não gera nenhum Javascript por padrão e é tão flexível quanto o Gatsby em fontes de dados, ainda que tenha menos ferramentas pra ele. Além desses, tem [MUITAS opções](https://www.staticgen.com/), pra todas as linguagens, necessidades e gostos.

Facilidade de postar ainda é um desafio, mas existem soluções: existem [diversos headless CMS](https://headlesscms.org/), como o [Netlify CMS](https://www.netlifycms.org/), feito pra SSGs e super prático; ou até o Wordpress, que expõe API e uma tem [source pronta pra Gatsby](https://www.npmjs.com/package/gatsby-source-wordpress).

Por fim, acho que super vale a pena pra um desenvolvedor web ter seu próprio site: nele você pode testar as últimas funcionalidades Javascript e CSS (Grid! :heart:) ou treinar acessibilidade no seu próprio tempo, sem precisar convencer chefe ou tomar decisões de produto arriscadas no trabalho.

**Comment by vhfmag** [comment]
> Acho que na web mesmo descentralizada sempre iremos precisar de algum tipo de agregador de conteúdos, porque há muita informação e por causa disso sempre vai haver algum tipo de "monopólio".

@AdrianoLupossa mas é justamente pra isso que servem padrões como RSS, Atom, Microsub e outros: pra definir uma forma interoperável de se inscrever e consumir feeds - ou seja, agregadores de conteúdo interoperáveis. Diria que seu ponto (ainda) é válido pro aspecto social da coisa: dar like, compartilhar e comentar posts... ([existem padrões pra isso](https://indieweb.org/), mas nós ainda não estamos lá – é difícil, custoso, desajeitado).

Só tenho certeza de que no *meu* feed, quero menos algoritmo e mais curadoria. Uma lista cuidadosamente podada com o tempo, sem apelação ou virulência (seja estilo Medium, como `Como nós aumentamos a nossa performance em 129782% (numa parte minúscula da nossa aplicação, mas dizer isso no título não geraria cliques)` e `7 coisas que todo desenvolvedor precisa pra ter um emprego`; ou estilo Twitter, como `Coisas que toda pessoa de sucesso faz` ou `O partido X come crianças`).

**Comment by kavalcante** [comment]
A facilidade em usar um agregador como o Medium em comparação ao RSS pelo menos pra mim é grande. Acho que de qualquer maneira, com essa debandada da plataforma, ela perde bastante poder. Porém, ter uma única plataforma e não depender de apps ou extensões que gerenciam RSS, pelo menos pra mim ainda é melhor.

**Comment by Diegocaetano23** [comment]
Realmente é o que comentaram acima, no Medium há bastante conteúdo bom, diversas opiniões e conteúdos de diversos lugares do planeta, com fácil acesso, no entanto, agora que é pago para acessar os conteúdos ilimitadamente, ficou mais difícil para algumas pessoas. Eu mesmo gostaria de continuar lendo os posts, mas pagar 50$ no momento está um pouco acima do meu orçamento! Triste! kkkkkkk Mas é um bom lugar para leitura em minha opinião!

**Comment by MechanicallyDev** [comment]
Alguém ainda acessa o medium sem janela anônima? 😂

Em ter., 4 de ago. de 2020 às 12:36, Diegocaetano23 <
notifications@github.com> escreveu:

**Comment by Diegocaetano23** [comment]
@MechanicallyDev Porque? O que muda acessar com janela anônima? Libera acesso? huhauhau n sei n, ajuda aew!

**Comment by MechanicallyDev** [comment]
Exatamente. Só abrir o artigo em janela anônima e consegue ler.

Em ter., 4 de ago. de 2020 às 15:27, Diegocaetano23 <
notifications@github.com> escreveu:

**Comment by Diegocaetano23** [comment]
ah entendi, verdade da pra acessar desse jeito msm kkk vlw.

**Comment by Diegocaetano23** [comment]
só q a janela anônima n fica mt bonita igual a original, mas o bom q acessa todo o conteúdo, msm dessa forma kk

**Comment by ninetails** [comment]
ainda curto acessar plataforma ao invés de RSS (preguiça de arrumar os feeds)... Apesar de recomendar a migração pro **dev.to**, acho o sistema de recomendação deles bem pouco relevante, também não há relevância em relação a bons conteúdos na hora do ranqueamento pra Home...
